alerts/openshift-virtualization-operator/LowVirtControllersCount.md

```diff
@@ -37,14 +37,14 @@ OpenShift Virtualization might become completely unresponsive.
 3. Check the `virt-launcher` logs for error messages:
 
    ```bash
-   $ oc -n $NAMESPACE logs <virt-launcher>
+   $ oc -n $NAMESPACE logs <virt-controller>
    ```
 
 4. Obtain the details of the `virt-launcher` pod to check for status conditions
 such as unexpected termination or a `NotReady` state.
 
    ```bash
-   $ oc -n $NAMESPACE describe pod/<virt-launcher>
+   $ oc -n $NAMESPACE describe pod/<virt-controller>
```

Comment:
```suggestion
   $ oc -n $NAMESPACE describe pods -l kubevirt.io=virt-controller
```
also, please fix the description (should be `virt-controller`, not `virt-launcher`)